### DISCUSSION TITLE
Only recent photos scanned for face recognition?

### DISCUSSION THREAD

**Discussion by yllekz** [photoview/photoview#228] [Q&A and Troubleshooting]
Hey there, I noticed a new "People" feature has shown up, and it already found a few faces but only in recent photos. Is there an option to tell it to go back and scan my library for face recognition?

**Comment by metbosch** [comment]
@yllekz Are you running using the docker-compose? Did you enabled the feature with some option? I don't see the "People" tab

**Reply by metbosch** [reply]
I just noticed that the `latest` docker images does not have the face recognition yet. I switched to `edge` tag and now the tab is there!

**Reply by yllekz** [reply]
I'm using docker-compose with the "edge" tag yeah. Actually you reminded me that I was still using that as opposed to latest. If it's still in development then I understand, and that explains things.

**Comment by viktorstrate** [comment]
@yllekz no there is no way to scan for new faces on existing images. You'd have to go to Settings, remove the filepath from the user and add it again, that will remove the media completely from the database and the cache, and rescan it from scratch.

**Reply by yllekz** [reply]
Hmm okay. I can give that a try...Is that a technical limitation? Just curious.

**Reply by viktorstrate** [reply]
It's not a technical limitation, but it would require some effort to support a smooth migration and I thought it wan't worth it.
